.github/workflows/image-factory.yaml

```diff
@@ -169,17 +169,11 @@ jobs:
   build-trillian:
     runs-on: ubuntu-20.04
     steps:
-      - name: Check out the remote rekor repository
-        uses: actions/checkout@v2
+      - name: Check out the remote trillian repository
+        uses: actions/checkout@v3
         with:
-          repository: securesign/rekor
-          path: rekor
-          ref: ${{ env.TRILLIAN_VER }}
-
-      - name: install ko
-        uses: imjasonh/setup-ko@v0.6
-        env:
-          KO_DOCKER_REPO: quay.io/securesign
+          repository: securesign/trillian
+          ref: release-next
```

Comment:
Once we have named/tracked branches in midstream, this should change to be something like `redhat-v1.5.2`.

Reply:
Ok so for now operating off of release-next is our m.o for the time being?